driver/pace/driver/run.py

```diff
@@ -324,7 +324,7 @@ def __init__(
             store=store, partitioner=partitioner, mpi_comm=comm
         )
 
-    def store(self, time: datetime, state: DriverState):
+    def store(self, time: datetime, state: DriverState, write_grid=False):
```

Comment:
`store`'s responsibilities become a lot more complicated with this new option, since state wants to be stored every step and grid wants to be stored only once. Can you move grid writing into its own method?

Reply:
Changed to `store_grid`